docs/docs/Components/components-memories.md

```diff
@@ -78,4 +78,24 @@ This component creates a `ZepChatMessageHistory` instance, enabling storage and
 
 | Name            | Type                    | Description                                           |
 |-----------------|-------------------------|-------------------------------------------------------|
-| message_history | BaseChatMessageHistory  | An instance of ZepChatMessageHistory for the session. |
\ No newline at end of file
+| message_history | BaseChatMessageHistory  | An instance of ZepChatMessageHistory for the session. |
+
+## GridGainChatMemory Component
+
+This component creates a chat message history using GridGain, enabling storage and retrieval of chat messages using GridGain's distributed caching capabilities.
+
+### Inputs
+
+| Name         | Type          | Description                                                           |
+|--------------|---------------|-----------------------------------------------------------------------|
+| host         | String        | GridGain server host address (default: "localhost"). Required.         |
```

Comment:
Lest put defaults at the end of these. For example, here - 
GridGain server host address (default: "localhost"). Required.  Default value: "localhost".